### PR TITLE
Users/harshit/app service controlupdate

### PR DESCRIPTION
**Pull request by HarshitArora8**
Please approve.

### CODE REVIEW

File: Control coverage/Feature/AppService.md [hunk 1]
```diff
@@ -17,6 +17,8 @@
 - [Azure_AppService_AuthZ_Configure_IP_Restrictions](#azure_appservice_authz_configure_ip_restrictions)
 - [Azure_AppService_AuthN_Use_Managed_Service_Identity](#azure_appservice_authn_use_managed_service_identity)
 
+- [Azure_AppService_DP_Dont_Use_FTP_Deployment](#azure_appService_dp_dont_use_ftp_deployment)
```
Comment (VamseeInala) [inline Control coverage/Feature/AppService.md:20]: @pranchalsomani The link doesn't seem to work, Can you please check?

File: Control coverage/Feature/AppService.md [hunk 2]
```diff
@@ -566,3 +568,51 @@ Managed Service Identity (MSI) allows your app to easily access other AAD-protec
...
+### Rationale 
+FTPS is used to enhance security for your Azure Web Application as it adds an extra layer of security to the FTP protocol,Enforcing FTPS-only Access for your Azure App Services apps can guarantee that the encrypted traffic between the web apps and servers and the FTP clients cannot be decrypted by malicious actors.
+
+### Control Spec 
+
+ >**Passed :**
+ > - FTP State found to be configured to FTPS (secure FTP) or marked as disabled for all slots in the App Service.
+ >
+> **Failed :**
+> - FTP State should either be configured to FTPS (secure FTP) or marked as disabled for all slots in the App Service.
```
Comment (VamseeInala) [inline Control coverage/Feature/AppService.md:585]: Failure criteria same as pass criteria?

File: Control coverage/Feature/AppService.md [hunk 3]
```diff
@@ -566,3 +568,51 @@ Managed Service Identity (MSI) allows your app to easily access other AAD-protec
...
+	 Go to Azure Portal --> your App Service --> Deployment --> Deployment slots --> Select slot --> Settings --> Configuration --> General Settings --> FTP state -->(Choose FTPS Only/Disabled based on requirement)--> Save.
+
+
+### Azure Policy or ARM API used for evaluation 
+
+- Azure Policy (built-in):
+  [Function apps should require FTPS only](https://ms.portal.azure.com/#view/Microsoft_Azure_Policy/PolicyDetailBlade/definitionId/%2Fproviders%2FMicrosoft.Authorization%2FpolicyDefinitions%2F399b2637-a50f-4f95-96f8-3a145476eb15)
+  <br />
+- Azure Policy (built-in):
+  [App Service apps should require FTPS only](https://ms.portal.azure.com/#view/Microsoft_Azure_Policy/PolicyDetailBlade/definitionId/%2Fproviders%2FMicrosoft.Authorization%2FpolicyDefinitions%2F4d24b6d4-5e53-4a4f-a7f4-618fa573ee4b)
```
Comment (VamseeInala) [inline Control coverage/Feature/AppService.md:606]: Why 2 times?

Comment (VamseeInala) [reply Control coverage/Feature/AppService.md:606]: If it is a different definition, Call out the difference clearly

**Review by VamseeInala — CHANGES_REQUESTED** [review]

File: Control coverage/Feature/AppService.md (hunk 3 above)
Comment (HarshitArora8) [reply Control coverage/Feature/AppService.md:606]: Resolving as these both are different MDC assessments.

File: Control coverage/Feature/AppService.md (hunk 1 above)
Comment (HarshitArora8) [reply Control coverage/Feature/AppService.md]: Link is working now.

File: Control coverage/Feature/AppService.md (hunk 2 above)
Comment (HarshitArora8) [reply Control coverage/Feature/AppService.md]: changed failing criteria.

**Review by VamseeInala — APPROVED** [review]

**Pull request merged**